test/Makefile

```diff
@@ -0,0 +1,78 @@
+comma := ,
+colon := :
+space :=
+space +=
+
+define colorecho
+	@tput setaf 6
+	@echo $1
+	@tput sgr0
+endef
+
+define capture-output
+	script $1 $2 > /dev/null || cat $1
+endef
+
+ROOT ?= ..
```

Comment:
I think the `ROOT` var here could use a more descriptive name, PACKAGE_ROOT / DM_JENKINS_ROOT maybe?

Reply:
Fair

Reply:
Renamed to `DM_JENKINS_REPO`